File: 2026-2/Basic/nohhj5/week2/방 번호.py
N = input()
numDict = {}

for n in N:
    if n not in numDict:
        numDict[n] = 1
    else:
        numDict[n] += 1

for i in '0123456789':
    if i not in numDict:
        numDict[i] = 0
sum6n9 = (numDict['6'] + numDict['9'])

if sum6n9 % 2 == 0:
    numDict['6'] = sum6n9 // 2
else:
    numDict['6'] = sum6n9 // 2 + 1
numDict['9'] = 0

maxVal = max(numDict.values())
print(maxVal)

File: 2026-2/Basic/hajaja1/Array · Linked List/방 번호.cpp
number = input() #숫자를 문자열으로 입력받는다
len_num = len(number) #숫자의 자릿수 새기
count_six_nine = 0
A = {}
for i in range(len_num):
    if number[i] in ("6", "9"):
        count_six_nine += 1
    else:
        A[number[i]] = A.get(number[i], 0) + 1
best = max(A.values(), default=0)
print(max(best, (count_six_nine + 1) // 2))
